### PR TITLE
T1916

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/72bc65abd37dd29/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/693fe3accd0ffb4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/72bc65abd37dd29/output.txt

Total script time: 0.64 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/693fe3accd0ffb4/output.txt

Total script time: 0.56 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/498f30c7dec410a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/498f30c7dec410a/output.txt

Total script time: 1.27 mins

Server is running on http://pr729.butyr.org/templates/basic/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/7ffe03d7413bf12/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/7ffe03d7413bf12/output.txt

Total script time: 1.21 mins

Server is running on http://pr729.butyr.org/templates/basic/

File: src/embed.js [hunk 1]
```diff
@@ -112,28 +112,49 @@ function init( window, document ) {
...
+      for( var i = 0; i < stateClasses.length; i++ ) {
+        el.classList.remove( stateClasses[ i ] );
+      }
+
+      el.classList.add( state );
+    }
+
+     $( "share-close" ).addEventListener( "click", function() {
       hide( "share" );
     }, false );
```
Comment (mjschranz) [inline src/embed.js:138]: This needs to be bumped over 2 spaces.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/480298775ae1ddf/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/480298775ae1ddf/output.txt

Total script time: 0.61 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/1efe6b5638b83c7/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/1efe6b5638b83c7/output.txt

Total script time: 0.85 mins

Server is running on http://pr729.butyr.org/templates/basic/

**Comment by jbuck** [comment]
Staged on master: https://github.com/mozilla/butter/commit/73e1ebb617d7ef749143f912fa18487ecf5d7809

**Pull request closed without merging**